src/cmd/package.go

```diff
@@ -262,13 +259,11 @@ func bindDeployFlags() {
 
 	v.SetDefault(V_PKG_DEPLOY_SET, map[string]string{})
 	v.SetDefault(V_PKG_DEPLOY_COMPONENTS, "")
-	v.SetDefault(V_PKG_DEPLOY_INSECURE, false)
 	v.SetDefault(V_PKG_DEPLOY_SHASUM, "")
 	v.SetDefault(V_PKG_DEPLOY_SGET, "")
 
 	deployFlags.StringToStringVar(&pkgConfig.DeployOpts.SetVariables, "set", v.GetStringMapString(V_PKG_DEPLOY_SET), "Specify deployment variables to set on the command line (KEY=value)")
 	deployFlags.StringVar(&pkgConfig.DeployOpts.Components, "components", v.GetString(V_PKG_DEPLOY_COMPONENTS), "Comma-separated list of components to install.  Adding this flag will skip the init prompts for which components to install")
-	deployFlags.BoolVar(&insecureDeploy, "insecure", v.GetBool(V_PKG_DEPLOY_INSECURE), "Skip shasum validation of remote package. Required if deploying a remote package and `--shasum` is not provided")
 	deployFlags.StringVar(&shasum, "shasum", v.GetString(V_PKG_DEPLOY_SHASUM), "Shasum of the package to deploy. Required if deploying a remote package and `--insecure` is not provided")
```

Comment:
This PR made me realize that the `shasum` variable we're setting this flag to isn't actually getting used. whoops..

Since the shasum flag is directly related to doing `--insecure` deployments, could you update this flag to be set to `pkgConfig.DeployOpts.Shasum`

Reply:
fixed in https://github.com/defenseunicorns/zarf/pull/1191/commits/4fb69aa44a9bd5dd894bca0ddc47711a25ced614